hbase_sql/src/test/java/com/splicemachine/derby/stream/spark/SparkDataSetTest.java

```diff
@@ -34,5 +49,29 @@ public SparkDataSetTest() {
     protected DataSet<ExecRow> getTenRowsTwoDuplicateRecordsDataSet() {
         return new SparkDataSet<>(SpliceSpark.getContext().parallelize(tenRowsTwoDuplicateRecords));
     }
+// Supported join types include: 'inner', 'outer', 'full', 'fullouter', 'leftouter', 'left', 'rightouter', 'right', 'leftsemi', 'leftanti'
+    @Test
+    public void testFoobar() {
```

Comment:
can we get a better test name? testFoobar doesn't really tell you much when the test fails.